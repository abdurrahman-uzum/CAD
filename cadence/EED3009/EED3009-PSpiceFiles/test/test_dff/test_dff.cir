** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 

** Creating circuit file "test_dff.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.INC "C:/Users/Abdurrahman/Desktop/cd4007/cd4007.lib" 
* From [PSPICE NETLIST] section of D:\cad\cadence\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 15ms 0 
.OPTIONS ADVCONV
.OPTIONS DIGINITSTATE= 0
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\test.net" 


.END
