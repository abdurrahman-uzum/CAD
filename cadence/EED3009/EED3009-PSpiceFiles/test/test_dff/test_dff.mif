 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\7400.lib, offset=2482, size=187
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\7400.lib, offset=43207, size=252
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\7400.lib, offset=2670, size=308
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\7400.lib, offset=43460, size=436
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\dig_io.lib, offset=34857, size=388
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\dig_io.lib, offset=36183, size=277
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\dig_io.lib, offset=7572, size=259
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\dig_io.lib, offset=106944, size=53
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\dig_io.lib, offset=106835, size=50
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\dig_io.lib, offset=38483, size=304
 lib=C:\Cadence\SPB_17.4\tools\PSpice\Library\dig_io.lib, offset=107059, size=193
