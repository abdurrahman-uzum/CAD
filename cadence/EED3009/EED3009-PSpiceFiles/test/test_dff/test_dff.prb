[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL NEVER
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
TEXT Time
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
BEGIN TRACE V(CLK)
MARKERID 2
TRACEADDEXT
END TRACE V(CLK)
END YAXIS 1
END ANAPLOT 1
BEGIN DIGPLOT 1
SIZE 14
MAXNAMELEN 8
TOP 0
CURRENT 0
PCT 33
BEGIN TRACE U52A:Q
MARKERID 1
TRACEADDEXT
END TRACE U52A:Q
END DIGPLOT 1
END DISPLAY LAST SESSION
