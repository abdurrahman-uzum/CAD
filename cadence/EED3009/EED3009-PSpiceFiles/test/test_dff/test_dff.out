
**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test_dff.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.INC "C:/Users/Abdurrahman/Desktop/cd4007/cd4007.lib" 
* From [PSPICE NETLIST] section of D:\cad\cadence\cdssetup\OrCAD_PSpice\17.4.0\PSpice.ini file:

**** INCLUDING cd4007.lib ****
.model nnMOS NMOS (LEVEL=2 VTo=1.4 Kp=.6m LAMBDA=0.005)
.model ppMOS PMOS (LEVEL=2 VTo=-1.0 KP=.6m LAMBDA=0.01)

**** RESUMING test_dff.cir ****
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 15ms 0 
.OPTIONS ADVCONV
.OPTIONS DIGINITSTATE= 0
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\test.net" 



**** INCLUDING test.net ****
* source EED3009
X_U53A         0 N58791 $G_DPWR $G_DGND 7404 PARAMS:
+ IO_LEVEL=0 MNTYMXDLY=0
V_V4         CLK 0  
+PULSE 0V 5V 0 1ns 1ns 0.5ms 1ms
X_U52A         N58791 N58705 CLK VDD N58753 N58705 VDD 0 7474 PARAMS:
+ IO_LEVEL=0 MNTYMXDLY=0
V_V5         VDD 0 5V

**** RESUMING test_dff.cir ****
.END


**** Generated AtoD and DtoA Interfaces ****

*
* Analog/Digital interface for node 0
*
* Moving X_U53A.U1:IN1 from analog node 0 to new digital node 0$AtoD
X$0_AtoD1
+ 0
+ 0$AtoD
+ $G_DPWR
+ $G_DGND
+ AtoD_STD
+       PARAMS: CAPACITANCE=   0     
*
* Analog/Digital interface for node CLK
*
* Moving X_U52A.UFF1:CLK from analog node CLK to new digital node CLK$AtoD
X$CLK_AtoD1
+ CLK
+ CLK$AtoD
+ VDD
+ 0
+ AtoD_STD
+       PARAMS: CAPACITANCE=   0     
*
* Analog/Digital interface for node VDD
*
* Moving X_U52A.UFF1:PREBAR from analog node VDD to new digital node VDD$AtoD
X$VDD_AtoD1
+ VDD
+ VDD$AtoD
+ VDD
+ 0
+ AtoD_STD
+       PARAMS: CAPACITANCE=   0     
*
* Analog/Digital interface power supply subcircuits
*
X$DIGIFPWR 0 DIGIFPWR


**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               D74CLMP         D74             
          IS    1.000000E-15  100.000000E-18 
          RS    2              25            
         CJO    2.000000E-12    2.000000E-12 


**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     BJT MODEL PARAMETERS


******************************************************************************




               Q74             
               NPN             
       LEVEL    1            
          IS  100.000000E-18 
          BF   49            
          NF    1            
         ISE  100.000000E-18 
          BR     .03         
          NR    1            
         ISC  400.000000E-18 
         ISS    0            
          RB   50            
          RE    0            
          RC   20            
         CJE    1.000000E-12 
         VJE     .9          
         MJE     .5          
         CJC  500.000000E-15 
         VJC     .8          
         MJC     .33         
        XCJC    1            
         CJS    3.000000E-12 
         VJS     .7          
         MJS     .33         
          TF  200.000000E-12 
          TR   10.000000E-09 
          KF    0            
          AF    1            
          CN    2.42         
           D     .87         


**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               nnMOS           ppMOS           
               NMOS            PMOS            
       LEVEL    2               2            
           L  100.000000E-06  100.000000E-06 
           W  100.000000E-06  100.000000E-06 
         VTO    1.4            -1            
          KP  600.000000E-06  600.000000E-06 
       GAMMA    0               0            
         PHI     .6              .6          
      LAMBDA    5.000000E-03     .01         
          IS   10.000000E-15   10.000000E-15 
          JS    0               0            
          PB     .8              .8          
        PBSW     .8              .8          
          CJ    0               0            
        CJSW    0               0            
        CGSO    0               0            
        CGDO    0               0            
        CGBO    0               0            
         TOX  100.000000E-09  100.000000E-09 
          XJ    0               0            
       UCRIT   10.000000E+03   10.000000E+03 
      DIOMOD    1               1            
         VFB    0               0            
        LETA    0               0            
        WETA    0               0            
          U0    0               0            
        TEMP    0               0            
         VDD    5               5            
       XPART    0               0            


**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     Digital Output MODEL PARAMETERS


******************************************************************************




               DO74            
    TIMESTEP  100.000000E-12 
      S0NAME X               
       S0VHI    2            
       S0VLO     .8          
      S1NAME 0               
       S1VHI     .8          
       S1VLO   -1.5          
      S2NAME R               
       S2VHI    1.4          
       S2VLO     .8          
      S3NAME R               
       S3VHI    2            
       S3VLO    1.3          
      S4NAME X               
       S4VHI    2            
       S4VLO     .8          
      S5NAME 1               
       S5VHI    7            
       S5VLO    2            
      S6NAME F               
       S6VHI    2            
       S6VLO    1.3          
      S7NAME F               
       S7VHI    1.4          
       S7VLO     .8          


**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     Digital Gate MODEL PARAMETERS


******************************************************************************




               D_04            
      TPLHMN    4.800000E-09 
      TPLHTY   12.000000E-09 
      TPLHMX   22.000000E-09 
      TPHLMN    3.200000E-09 
      TPHLTY    8.000000E-09 
      TPHLMX   15.000000E-09 


**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     Digital Edge Triggered FF MODEL PARAMETERS


******************************************************************************




               D_74            
  TPCLKQLHMN    5.600000E-09 
  TPCLKQLHTY   14.000000E-09 
  TPCLKQLHMX   25.000000E-09 
  TPCLKQHLMN    8.000000E-09 
  TPCLKQHLTY   20.000000E-09 
  TPCLKQHLMX   40.000000E-09 
   TPPCQLHMN    6.250000E-09 
   TPPCQLHTY   15.625000E-09 
   TPPCQLHMX   25.000000E-09 
   TPPCQHLMN   10.000000E-09 
   TPPCQHLTY   25.000000E-09 
   TPPCQHLMX   40.000000E-09 
    TWCLKLMN   37.000000E-09 
    TWCLKLTY   37.000000E-09 
    TWCLKLMX   37.000000E-09 
    TWCLKHMN   30.000000E-09 
    TWCLKHTY   30.000000E-09 
    TWCLKHMX   30.000000E-09 
     TWPCLMN   30.000000E-09 
     TWPCLTY   30.000000E-09 
     TWPCLMX   30.000000E-09 
   TSUDCLKMN   20.000000E-09 
   TSUDCLKTY   20.000000E-09 
   TSUDCLKMX   20.000000E-09 
 TSUPCCLKHMN    0            
 TSUPCCLKHTY    0            
 TSUPCCLKHMX    0            
    THDCLKMN    5.000000E-09 
    THDCLKTY    5.000000E-09 
    THDCLKMX    5.000000E-09 
  TSUCECLKMN    0            
  TSUCECLKTY    0            
  TSUCECLKMX    0            
   THCECLKMN    0            
   THCECLKTY    0            
   THCECLKMX    0            


**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     Digital IO  MODEL PARAMETERS


******************************************************************************




               IO_STD          
        DRVL  104            
        DRVH   96.4          
       AtoD1 AtoD_STD        
       AtoD2 AtoD_STD_NX     
       AtoD3 AtoD_STD        
       AtoD4 AtoD_STD_NX     
       DtoA1 DtoA_STD        
       DtoA2 DtoA_STD        
       DtoA3 DtoA_STD        
       DtoA4 DtoA_STD        
      TSWHL1    1.511000E-09 
      TSWHL2    1.487000E-09 
      TSWHL3    1.511000E-09 
      TSWHL4    1.487000E-09 
      TSWLH1    3.517000E-09 
      TSWLH2    3.564000E-09 
      TSWLH3    3.517000E-09 
      TSWLH4    3.564000E-09 
       TPWRT  100.000000E+03 


**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  CLK)    0.0000  (  VDD)    5.0000  ($G_DGND)    0.0000                       

($G_DPWR)    5.0000                   (X$0_AtoD1.1)     .0915                   

(X$0_AtoD1.2)     .0457               (X$0_AtoD1.3)     .8277                   

(X$CLK_AtoD1.1)     .0915             (X$CLK_AtoD1.2)     .0457                 

(X$CLK_AtoD1.3)     .8277             (X$VDD_AtoD1.1)    1.5648                 

(X$VDD_AtoD1.2)     .7824             (X$VDD_AtoD1.3)    2.2862             



 DGTL NODE : STATE  DGTL NODE : STATE  DGTL NODE : STATE  DGTL NODE : STATE


(  N58753) : 0     (CLK$AtoD) : 0     (VDD$AtoD) : 1     (  N58791) : 1         

(  N58705) : 1     (  0$AtoD) : 0     




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V4         1.043E-03
    V_V5        -1.742E-03
    X$DIGIFPWR.VDPWR  -1.048E-03
    X$DIGIFPWR.VDGND  -1.048E-03

    TOTAL POWER DISSIPATION   1.40E-02  WATTS



          JOB CONCLUDED

**** 11/30/22 15:59:08 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "test-test_dff"  [ D:\cad\cadence\EED3009-PSpiceFiles\test\test_dff.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  License check-out time            =         .88
  Total job time (using Solver 1)   =         .06
